Updating pipeline docs for workshop (#550)
aicoe-aiops/ocp-ci-analysis: docs/workshop/ml_pipeline.md

# Create an ML Pipeline using Elyra and Kubeflow Pipelines

> 

Reviewer: In this section, we will learn how to run the training pipeline of the ML model in automation.

To learn more about the training process refer to the chapter on [model development](./model_development.md).
```suggestion

```

Reviewer: can you include the above to the introduction. We are going into creating the pipeline very abruptly

> In order to run the above notebooks in automation, we can configure a Kubeflow Pipeline using the Elyra notebook pipelines editor from Jupyterlab.

Reviewer: ```suggestion
In order to run the above notebooks in automation, we use Kubeflow Pipelines and the [Elyra notebook pipelines editor](https://elyra.readthedocs.io/en/stable/) from Jupyterlab.
```
```suggestion
In order to run the above notebooks in automation, we can configure a Kubeflow Pipeline using the Elyra notebook pipelines editor from Jupyterlab.https://www.kubeflow.org/docs/components/pipelines/introduction/
```

## Create kubeflow runtime configuration

> 

Reviewer: @suppathak can correct the link to the pipeline Open pipeline(end-to-end.pipeline) ?
